From 0b55abd34a2e9ddc6a0cad51a8a292791e2c7fbb Mon Sep 17 00:00:00 2001
From: Masahiro Yamada <yamada.masahiro@socionext.com>
Date: Tue, 17 Oct 2017 21:19:40 +0900
Subject: [PATCH 017/105] ARM: uniphier: enable DWC3 xHCI driver really

I thought commit d37d31849c6a ("ARM: uniphier: enable DWC3 xHCI
driver") enabled CONFIG_USB_DWC3_UNIPHIER, but CONFIG_USB_XHCI_DWC3
was missing in uniphier_v7_defconfig.  Re-add.

Signed-off-by: Masahiro Yamada <yamada.masahiro@socionext.com>
---
 configs/uniphier_v7_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/uniphier_v7_defconfig b/configs/uniphier_v7_defconfig
index 5c1d343691..b4b54c06bd 100644
--- a/configs/uniphier_v7_defconfig
+++ b/configs/uniphier_v7_defconfig
@@ -49,7 +49,9 @@ CONFIG_SMC911X_BASE=0x0
 CONFIG_SMC911X_32_BIT=y
 CONFIG_USB=y
 CONFIG_USB_XHCI_HCD=y
+CONFIG_USB_XHCI_DWC3=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_EHCI_GENERIC=y
 CONFIG_USB_DWC3=y
+CONFIG_USB_DWC3_UNIPHIER=y
 CONFIG_USB_STORAGE=y
-- 
2.13.6

